go.mod

```diff
@@ -5,48 +5,45 @@ go 1.14
 replace (
 	github.com/hashicorp/consul v0.0.0-20171026175957-610f3c86a089 => github.com/hashicorp/consul/sdk v0.7.0
 	github.com/hashicorp/consul/api v1.3.0 => github.com/hashicorp/consul/api v0.0.0-20191112221531-8742361660b6
+	github.com/kyma-incubator/hydroform/function => github.com/pPrecel/hydroform/function v0.0.0-20210206085647-39a02f919fe9
 	// this is needed for terraform to work with the k8s 0.18 APIs, we should be able to remove it once we have terraform 0.13+
 	github.com/terraform-providers/terraform-provider-openstack => github.com/terraform-providers/terraform-provider-openstack v1.20.0
 	// grpc need to be compatible with direct dependencies in terraform (>=v1.29.1)
 	google.golang.org/grpc => google.golang.org/grpc v1.29.1
 	k8s.io/client-go => k8s.io/client-go v0.18.9
+
 )
 
 require (
-	github.com/Microsoft/go-winio v0.4.15 // indirect
-	github.com/Microsoft/hcsshim v0.8.10 // indirect
 	github.com/avast/retry-go v2.6.1+incompatible
 	github.com/blang/semver/v4 v4.0.0
 	github.com/briandowns/spinner v1.12.0
 	github.com/containerd/cgroups v0.0.0-20201119153540-4cbc285b3327 // indirect
-	github.com/containerd/containerd v1.4.2 // indirect
-	github.com/containerd/continuity v0.0.0-20200710164510-efbc4488d8fe // indirect
 	github.com/coreos/etcd v3.3.13+incompatible // indirect
 	github.com/daviddengcn/go-colortext v1.0.0
-	github.com/docker/cli v0.0.0-20200130152716-5d0cf8839492
-	github.com/docker/docker v1.4.2-0.20200203170920-46ec8731fbce
+	github.com/docker/cli v20.10.3+incompatible
```

Comment:
Both docker dependencies are heavily used by the `kyma install` command when installing from local sources into a remote cluster.
Did you test that it still works?

Reply:
Good question. Maybe first I should try to migrate to the previous version of these dependencies

Reply:
It is fine to upgrade them, but we need to be sure nothing breaks.

Reply:
Ok I will check this ;)

Reply:
I tested provisioning and installation. Everything works for me